terra2/README.md

```diff
@@ -0,0 +1,136 @@
+# Terra Wormhole Contract Deployment
+
+This readme describes the steps for building, verifying, and deploying Terra smart contracts for Wormhole.
+
+**WARNING**: *This process is only Linux host compatible at this time.*
+
+## Verify Tilt
+
+Before building Terra contracts, ensure that the specific commit you will be
+building from passes in tilt. This that ensures basic functionality of the
+Terra smart contracts that you are about to build and deploy.
+
+## Build Contracts
+
+The following command can be used to build Terra contracts via Docker.
+
+Build Target Options: [`mainnet`|`testnet`|`devnet`|
+
+These network names correspond to the naming convention used by wormhole
+elsewhere. This means that `mainnet` corresponds to Terra `mainnet`,
+`testnet` corresponds to Terra `testnet`, and `devnet` is `localterra`.
+
+```console
+wormhole/terra $ make artifacts
```

Comment:
nit: "wormhole/terra2" 

Same for all the ones below.

Reply:
fixed